src/main/java/net/minecraftforge/client/event/ScreenEvent.java

```diff
@@ -290,50 +273,70 @@ public PoseStack getPoseStack()
     }
 
     /**
-     * Fired to determine whether to render the potion indicators in the {@link EffectRenderingInventoryScreen inventory
-     * screen} in compact or classic mode.
+     * Fired ahead of rendering any active mob effects in the {@link EffectRenderingInventoryScreen inventory screen}.<p/>
+     * Can be used to select the size of the effects display (full or compact) or even hide or replace vanilla's rendering entirely.
      *
-     * <p>This event is not {@linkplain Cancelable cancellable} and {@linkplain HasResult has a result}. </p>
-     * <ul>
-     *   <li>{@link Result#ALLOW} - forcibly renders the potion indicators in <em>compact</em> mode.</li>
-     *   <li>{@link Result#DEFAULT} - defaults to vanilla behavior to using compact mode if the the screen width is too
-     *   small for classic rendering of potion indicators.</li>
-     *   <li>{@link Result#DENY} - forcibly renders the potion indicators in <em>classic</em> mode.</li>
-     * </ul>
+     * <p>This event is {@linkplain Cancelable cancellable} and does not {@linkplain HasResult have a result}.</p>
+     * Cancelling this event will prevent vanilla rendering.
```

Comment:
The closing paragraph tag should be after the sentence describing what cancelling the event does.
```suggestion
     * <p>This event is {@linkplain Cancelable cancellable} and does not {@linkplain HasResult have a result}.
     * Cancelling this event will prevent vanilla rendering.</p>
```